client/src/components/Donation/donation-text-components.tsx

```diff
@@ -18,9 +18,9 @@ export const DonationText = (): JSX.Element => {
   const { t } = useTranslation();
   return (
     <>
-      <p>{t('donate.efficiency')}</p>
-      <p>{t('donate.why-donate-1')}</p>
-      <p>{t('donate.why-donate-2')}</p>
+      <p data-test-label='donateEfficiencyP1'>{t('donate.efficiency')}</p>
```

Comment:
@mirrzamustafa Lets try and avoid adding these labels.

What we are doing here, is to draw a side by side comparison of cypress and playwright as a tool choice. If this file was tested using cypress, without adding these test-labels, it speaks loads about the ability and power playwright holds as a tool.

I therefore recommend we do not change the code we need to test.